connectors/src/lib/models/webcrawler.ts

```diff
@@ -57,7 +57,7 @@ WebCrawlerConfiguration.init(
     },
     depth: {
       type: DataTypes.INTEGER,
-      allowNull: true,
+      allowNull: false,
```

Comment:
Will be able to perform this migration in prod? I mean does all the webcrawler configurations have a depth in production?

Reply:
I will manually update them before.